ansible/files/permission_check.py

```diff
@@ -9,18 +11,18 @@
         {"groupname": "ssl-cert", "username": "postgres"}
     ],
     "ubuntu": [
-        {"groupname":"ubuntu","username":"ubuntu"},
-        {"groupname":"adm","username":"ubuntu"},
-        {"groupname":"dialout","username":"ubuntu"},
-        {"groupname":"cdrom","username":"ubuntu"},
-        {"groupname":"floppy","username":"ubuntu"},
-        {"groupname":"sudo","username":"ubuntu"},
-        {"groupname":"audio","username":"ubuntu"},
-        {"groupname":"dip","username":"ubuntu"},
-        {"groupname":"video","username":"ubuntu"},
-        {"groupname":"plugdev","username":"ubuntu"},
-        {"groupname":"lxd","username":"ubuntu"},
-        {"groupname":"netdev","username":"ubuntu"}
+        {'groupname': 'adm', 'username': 'ubuntu'},
+        {'groupname': 'audio', 'username': 'ubuntu'},
+        {'groupname': 'cdrom', 'username': 'ubuntu'},
+        {'groupname': 'dialout', 'username': 'ubuntu'},
+        {'groupname': 'dip', 'username': 'ubuntu'},
+        {'groupname': 'floppy', 'username': 'ubuntu'},
+        {'groupname': 'lxd', 'username': 'ubuntu'},
+        {'groupname': 'netdev', 'username': 'ubuntu'},
+        {'groupname': 'plugdev', 'username': 'ubuntu'},
+        {'groupname': 'sudo', 'username': 'ubuntu'},
+        {'groupname': 'ubuntu', 'username': 'ubuntu'},
+        {'groupname': 'video', 'username': 'ubuntu'}
```

Comment:
Nit: inconsistent single/double quotes and spacing? But alphabetized 💚

Reply:
Good point - ran both this and the shell script through an autoformatter.